### ISSUE TITLE
Video not centered in player

### ISSUE THREAD

**Issue by Kaneco** [YePpHa/YouTubeCenter#1361]
So I've had this issue for ages now that when I view a video its not centered on the player unless I maximize the player, so I can never see the whole video unless I maximize the player.

I have gone through the settings numerous times, etc, etc... Already reinstalled the script and never managed to solve it, would appreciate some help.

**Comment by Eisys** [comment]
What version of YTC are you using? This is a really old issue and if you are not using the latest DEV version of YTC, do so, it's fixed there!
https://github.com/YePpHa/YouTubeCenter/wiki/Developer-Version

**Comment by tbs** [comment]
This problem is back few days ago, I already using the dev version (build 461), as always.

**Comment by Long-Horse** [comment]
Yeah this problem is back in 461. Started a couple days ago. @Eisys https://gfycat.com/FeminineAgreeableBirdofparadise

**Comment by RedDeadElite** [comment]
Go to Player>>Player Size and make sure that Only Stage Mode is ticked.

**Comment by ireun** [comment]
@RedDeadElite That worked for me.. :)

**Comment by Owyn** [comment]
@RedDeadElite That worked for me too :)

... so there won't be a "fix" ?

**Comment by ansaksie** [comment]
If this is the same problem I'm thinking of, I have it, too; the video itself doesn't resize with the player, so you have a large video in a small player.  Only Stage Mode works because it's a larger player, I think.  If you want a smaller player, I've found that the video will resize and center correctly in Flash and Aggressive Flash player types.  It's the HTML5 and Default player types that need to be fixed.
